pkg/dbtoken/server_test.go

```diff
@@ -64,7 +64,7 @@ func TestServer(t *testing.T) {
 					Path:   "/token",
 				},
 			},
-			wantStatusCode: http.StatusMethodNotAllowed,
```

Comment:
This is changing because previously we set a route only for POST on the `/token` endpoint, and now using the stdlib mux it handles all HTTP method requests for the endpoint correct?  So it's hitting the auth endpoint which is why we're returning Forbidden now instead of MethodNotAllowed.  

If so, can we add a unit test case for an authenticated non-GET request to cover the new introduction?
```go
	if r.Method != http.MethodPost {
		http.Error(w, "method not allowed", http.StatusMethodNotAllowed)
	}
```

Reply:
done